Configurations/10-main.conf

```diff
@@ -192,7 +192,7 @@ my %targets = (
         CFLAGS           => picker(default => "-Wall",
                                    debug   => "-O0 -g",
                                    release => "-O3"),
-        cppflags         => "-D_POSIX_C_SOURCE=200112L -D_BSD -D_VOS_EXTENDED_NAMES -DB_ENDIAN",
+        lib_cppflags     => "-D_POSIX_C_SOURCE=200112L -D_BSD -D_VOS_EXTENDED_NAMES -DB_ENDIAN",
```

Comment:
I wouldn't be surprised if non-B_ENDIAN belong in cppflags. Indeed, it probably affects POSIX-ness, and apps do include POSIX headers.

Reply:
Fixed